### DISCUSSION TITLE
Repo address

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [DominiqueMakowski/teaching#17] [Show and tell]
paste it here

**Comment by rejoha** [comment]
https://github.com/rejoha/foradvancedpythonclass

**Comment by MartynaPlomecka** [comment]
https://github.com/MartynaPlomecka/testrepo

**Comment by Cymelo92** [comment]
https://github.com/Cymelo92/testrepo

**Comment by Pascal-Kueng** [comment]
https://github.com/Pascal-Kueng/testrepo

**Comment by nriese** [comment]
https://github.com/nriese/PNAS_Paper

**Comment by NimsajB** [comment]
https://github.com/NimsajB/testrepo

**Comment by AnaRodMa** [comment]
https://github.com/AnaRodMa/trestrepo

**Comment by GeorgiaSou** [comment]
https://github.com/GeorgiaSou/testrepo

**Comment by Becca495** [comment]
https://github.com/Becca495/Rebecca_testrepo

**Comment by zitamayer** [comment]
https://github.com/zitamayer/testrepo

**Comment by DominiqueMakowski** [comment]
Thanks all and good luck with the road ahead 💪 

@zitamayer @Becca495 @GeorgiaSou @MartynaPlomecka @Cymelo92 @AnaRodMa @Pascal-Kueng @NimsajB @nriese @Becca495 @rejoha @LenaSchn @SvennoNito
